tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I've added this for testing in VSCode which is now run at the root directory

Reply:
I'm worried this will cause issues or confusion with our existing 'root' tsconfig file here: https://github.com/celo-org/celo-monorepo/blob/master/packages/typescript/tsconfig.json#L1